tic_tac_toe/game.py

```diff
@@ -87,18 +130,47 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
 
+    if game['winner'] or _board_is_full(game['board']):
+        raise InvalidMovement("Game is over.")
+    
+    elif get_next_turn(game) != player:
+        raise InvalidMovement('"{}" moves next'.format(game['next_turn']))
+    if _position_is_valid(position):
+        if not _position_is_empty_in_board(position, game['board']):
+            raise InvalidMovement("Position already taken.")
+        game['board'][position[0]][position[1]] = player
+        if game['next_turn'] == game['player1']:
+            game['next_turn'] = game['player2']
+        else:
+            game['next_turn'] = game['player1']
```

Comment:
Here could have been a great place for a one line `if` statement:
`game['next_turn'] = game['player2'] if game['next_turn'] == game['player1'] else game['player1']`